v2/cmd/wails/internal/gomod/gomod.go

```diff
@@ -32,12 +34,13 @@ func SyncGoMod(logger *clilogger.CLILogger, updateWailsVersion bool) error {
 		LogGreen("Updated go.mod to use Go '%s'", goversion.MinRequirement)
 	}
 
-	if outOfSync, err := gomod.GoModOutOfSync(gomodData, internal.Version); err != nil {
+	internalVersion := strings.TrimSpace(internal.Version)
+	if outOfSync, err := gomod.GoModOutOfSync(gomodData, strings.TrimSpace(internalVersion)); err != nil {
```

Comment:
Just a minor comment, seem like a copy paste error 😄 
```suggestion
	if outOfSync, err := gomod.GoModOutOfSync(gomodData, internalVersion); err != nil {
```

Reply:
Indeed -- oops. Thanks!